articles/tutorials/qft-tutorial.md

```diff
@@ -231,34 +294,61 @@ namespace Quantum.Operations {
 }
 ```
 
+Note that we added an extra line, `@EntryPoint()` before the operation. This is
+a label to point the compiler the entry point to start the computation.
 
-With the Q# file and operation complete, it is ready to be called and simulated by the classical host.
+With the Q# file and operation complete we can save it. Now it is ready to be
+simulated by executing the file from the command line.
 
-## Set up the classical host
+## Execute the program
 
-Having defined our Q# operation in a `.qs` file, we now write the classical host program to call that operation and process any returned classical data (for now, there isn't anything to process---recall that our operation defined above returns `Unit`).
-When we later modify the Q# operation to return an array of measurement results (`Result[]`), we will update the host file correspondingly.
+To execute open the terminal in the folder of your project and enter:
 
-While the Q# program is ubiquitous across classical host languages, the host programs of course will vary slightly. As such, please follow the instructions in the tab corresponding to your development environment.
+```dotnetcli
+dotnet run
+```
 
-#### [Python](#tab/tabid-python)
+<!-- ## Set up the classical host
```

Comment:
Let's just delete this instead of commenting it out, especially since it's still easy to look up in the commit history.

Reply:
Sure! This was just to help Gillen decide in case he wanted to keep everything!